python/smashbox/utilities/__init__.py

```diff
@@ -228,23 +232,32 @@ def oc_webdav_url(protocol='http',remote_folder="",user_num=None,webdav_endpoint
 
     # strip-off any leading / characters to prevent 1) abspath result from the join below, 2) double // and alike...
     remote_folder = remote_folder.lstrip('/')
-
+    
     if webdav_endpoint is None:
         webdav_endpoint = config.oc_webdav_endpoint
-
+        
     remote_path = os.path.join(webdav_endpoint, config.oc_server_folder, remote_folder)
 
     if user_num is None:
         username = "%s" % config.oc_account_name
     else:
         username = "%s%i" % (config.oc_account_name, user_num)
-
-    if hide_password:
+    
+    if hide_password== True:
         password = "***"
     else:
         password = config.oc_account_password
-
-    return protocol + '://' + username + ':' + password + '@' + config.oc_server + '/' + remote_path
+    if local_folder==None:
```

Comment:
Is it working for you? Will it be compatibile with other operating systems? What about having smashbox on virtual machine in python, so that it will be compatibile with Windows/Mac/Linux users?